### PR TITLE
Multirobot Detector initial implementation

### PR DESCRIPTION
**Pull request by Mitchell-Conrad-20**
Detector provides a list of robots with an active LOS and a list of robots that have been lost.

### CODE REVIEW

File: .gitignore
Comment (stoneoxmike) [inline .gitignore:1]: Based

File: OpticalCommunications-2023-N-robots-TCP/python_implementation/threads/detector_manager.py
Comment (stoneoxmike) [inline OpticalCommunications-2023-N-robots-TCP/python_implementation/threads/detector_manager.py:1]: I thought I fixed this already lol

**Review by stoneoxmike — APPROVED** [review]
Everything looks good to me

File: OpticalCommunications-2023-N-robots-TCP/python_implementation/threads/detector_manager.py
Comment (Mitchell-Conrad-20) [reply OpticalCommunications-2023-N-robots-TCP/python_implementation/threads/detector_manager.py:1]: I didn't modify detector_manager. I think someone must've changed something on dev which rolled back a few of the changes. This branch must've been made before that happened, so the changes were preserved. Same thing with the .gitignore, I didn't modify that

**Review by bhageneder — APPROVED** [review]

**Pull request merged**